### DISCUSSION TITLE
v0.8.0

### DISCUSSION THREAD

**Discussion by timothyschoen** [plugdata-team/plugdata#1206] [General]
Huge update!

- Updated to pd-0.54 with multichannel signals!
- Updated to ELSE-rc9 with many new objects and multi-channel support
- New logo by [Bas de Bruin](https://www.bdebruin.nl/)
- UI improvements
    - New add objects menu
    - Redesigned palette bar
    - Redesigned settings panel
    - Redesigned inspector
    - Redesigned deken
    - Redesigned Heavy dialog
    - Redesigned save dialog
    - Redesigned level meter
    - Redesigned object reference panel
    - Improved tab and palette drag and drop system
    - Fixed small UI alignment details everywhere
- New activity overlay will highlight objects that are generating messages
- New alignment options allow you to align or space objects evenly
- DAW parameters can now be float, int, exponential float or logarithmic float
- Many stability improvements
	- New systems are in place that make plugdata a lot more thread-safe
- plugdata's folder is now always in ~/Documents/plugdata, so location is no longer OS-specific
- Improvements to Heavy toolchain
- Improvements to object compatibility with pure-data, ELSE and cyclone.
- Fixed GraphOnParent size inconsistencies
- Small improvements to theming
- Deleting a folder in search path no longer causes crash
- PdControl can now open links
- Implemented "menunew" and "open" messages
- Fix broken encapsulate function
- Can now choose to send MIDI to a specific device in standalone, instead of always sending to all devices
- Fixed "vis" message to clone
- Fixed listbox sending output when created
- Fixed crash in Logic when loading a project with parameters
- Fixed various issues with $0 expansion in send/receive symbols
- Fixed weird output gain when enabling protected mode
- Connections are now locked in temp lock mode too
- Improved auto-patching behaviour
- Fixed many messages to pd objects not working
- And much more!

Thanks to @alcomposer @tomara-x @agraef @dromer @vasilymilovidov @caitp @haenkel for contributing. And thanks to everyone who reported bugs, thanks to you, plugdata is now more stable than ever!

<hr /><em>This discussion was created from the release <a href='https://github.com/plugdata-team/plugdata/releases/tag/'>v0.8.0</a>.</em>

**Comment by polarity** [comment]
nice work team! will have play with this tomorrow.

**Comment by mngyuan** [comment]
The right click menu now lacks Add->, and it's been moved to the new UI, which I find a lot less usable for having a complete reference that's quick to use. Is it possible to return this functionality to the right click menu?

**Reply by timothyschoen** [reply]
Yeah we could do this, more people have asked. I personally didn't like how the old menu was this huge popup, where a large part of the objects were hidden inside a submenu, but I'll consider bringing it back.

The idea of the new menu is that it both shows more options without needing submenus (24 now vs 8 before), and allows you to find stuff more quickly by recognizing icons (instead of having to read all the text).
